umap/static/umap/js/umap.js

```diff
@@ -250,7 +250,18 @@ L.U.Map.include({
         }
         this._controls.zoom = new L.Control.Zoom({zoomInTitle: L._('Zoom in'), zoomOutTitle: L._('Zoom out')});
         this._controls.datalayers = new L.U.DataLayersControl(this);
-        this._controls.locate = new L.U.LocateControl();
+        this._controls.locate = L.control.locate({
+            strings: {
+                title: L._('Center map on your location'),
+            },
+            showPopup: false,
+            // We style this control in our own CSS for consistency with other controls,
+            // but the control breaks if we don't specify a class here, so a fake class
+            // will do.
+            icon: 'umap-empty',
```

Comment:
Watch out, there is a `umap-empty` css [class](https://github.com/umap-project/umap/blob/7893ff1c7d943b7fb71b74dbd8a3b47b7b104ca3/umap/static/umap/base.css#L471) ;)

Reply:
Oh! I really should have checked :sweat_smile: